api/v1/storagecluster_types.go

```diff
@@ -185,11 +185,13 @@ type ManageCephDashboard struct {
 	SSL bool `json:"ssl,omitempty"`
 }
 
-// ManageCephBlockPools defines how to reconcilea CephBlockPools
+// ManageCephBlockPools defines how to reconcile CephBlockPools
 type ManageCephBlockPools struct {
 	ReconcileStrategy    string `json:"reconcileStrategy,omitempty"`
 	DisableStorageClass  bool   `json:"disableStorageClass,omitempty"`
 	DisableSnapshotClass bool   `json:"disableSnapshotClass,omitempty"`
+	// if set to true, the storageClass created for cephBlockPools will be annotated as the default for the whole cluster
+	DefaultSCForCluster bool `json:"defaultSCForCluster,omitempty"`
```

Comment:
`DefaultStrageClass` sounds good to keep the SC-related members naming convention similar?

Reply:
done